src/utils/mediaQueries.ts

```diff
@@ -1,30 +1,38 @@
 import { Command } from 'types'
 
+export enum TypeMediaQueries {
+  XL = 'xl',
+  LG = 'lg',
+  MD = 'md',
+  SM = 'sm',
+  XS = 'xs',
+}
+
```

Comment:
what about

```js
export type Breakpoints = 'xl' | 'lg' | 'md' | 'sm' | 'xs'
```

so you don't have to import the enum?

Reply:
:thinking: